lib/xandra/protocol.ex

```diff
@@ -407,7 +427,8 @@ defmodule Xandra.Protocol do
   end
 
   defp decode_error_message(_reason, buffer) do
-    {message, _buffer} = decode_string(buffer)
+    decode_string(buffer, message)
+    _ = buffer
```

Comment:
I think we can improve this as it's a bit "hacky" to do this to ignore unused variable warnings (I assume that's why we do it). For example, we could have `decode_string(buffer, message, reuse_buffer: false)` or something along these lines. Thoughts?